src/Server_Command.php

```diff
@@ -97,9 +98,9 @@ function __invoke( $_, $assoc_args ) {
 		// https://bugs.php.net/bug.php?id=60181
 		$options = array();
 		if ( \WP_CLI\Utils\is_windows() ) {
```

Comment:
Use an import for `WP_CLI\Utils`:
```suggestion
		if ( Utils\is_windows() ) {
```